.github/workflows/test_patch.yml

```diff
@@ -0,0 +1,44 @@
+name: ONNX / Python - Test
```

Comment:
please use a different name, we already have a workflow with this name so it's confusing.